examples/opensearchserverless/main.tf

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+# Require Terraform 0.12 or greater
+terraform {
+  required_version = ">= 0.12"
+}
+
+# Set AWS provider region
+provider "aws" {
+  region = var.aws_region
+}
+
+# Creates an encryption security policy
```

Comment:
We don't say encryption security policy, rather we say encryption policy in our docs, Can you change it to that.